.github/workflows/prcop-config.json

```diff
@@ -3,7 +3,7 @@
     {
       "name": "descriptionRegexp",
       "config": {
-        "regexp": "x] Testing instructions",
+        "regexp": "(?i)x] Testing instructions",
```

Comment:
@unlikelyzero This was tripping me up as I was using `X` to check the boxes in the template instead of `x`

Reply:
thanks for fixing this. You're not the only one who was using the INCORRECT markdown format ;)